### PR TITLE
LPS-117071 Extra characters are inserted in an integer field after saving

### PR DESCRIPTION
**Pull request by natocesarrego**
Hi @natocesarrego,
Could you please review this pull request?
I'm reverting this commit because it cause regression. The Integer type is a Number type, so we can't separate the conditions. Or maybe we can swap the two conditions, but I'm not sure we need to format the integer value.
Original commits: https://github.com/brianchandotcom/liferay-portal/pull/83838 
Thank you and best regards,
Marci

### CODE REVIEW

**Comment by MarcellGyopos** [comment]
Hi @leoadb 
Could you please check this pull request?
Thank you.

**Comment by leoadb** [comment]
@MarcellGyopos we can't simply revert it because a integer number will display a decimal separator. Can you please try a solution that works with both scenarios? Thanks!

**Comment by MarcellGyopos** [comment]
Hi @leoadb ,
I'm not exactly sure what are you pointing for.
Could you please explain it?
Thank you.

**Comment by MarcellGyopos** [comment]
If I revert, the integer will not be formatted in the future.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/FieldsToDDMFormValuesConverterImpl.java [hunk 1]
```diff
@@ -211,19 +211,18 @@ protected String getDDMFieldValueString(
 
 			fieldValue = valueDate.getTime();
 		}
-		else if (fieldValue instanceof Number) {
+		else if ((fieldValue instanceof Number) &&
+				 !(fieldValue instanceof Integer)) {
```
Comment (leoadb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/FieldsToDDMFormValuesConverterImpl.java:215]: @MarcellGyopos See this line. If you remove **!(fieldValue instanceof Integer)** from the condition then in line 225 an integer number will be formatted as a double including the decimal separator and a zero.

**Comment by leoadb** [comment]
@MarcellGyopos you are right. Sorry I messed the things up :)

**Comment by leoadb** [comment]
ci:forward

**Comment by leoadb** [comment]
ci:reopen

**Comment by MarcellGyopos** [comment]
Thank you:)

**Pull request closed without merging**